packages/graphql/fragments/postFieldsFragment.js

```diff
@@ -12,6 +13,30 @@ export default gql`
       id
       name
       avatarUrl
+      groupRoles {
```

Comment:
this is a lot of new data to load with every post and will slow things down. why do we need this?